client/js/app.js

```diff
@@ -13,6 +13,36 @@ function loadJSON(endpoint, type, body, callback, error) {
     }
 }
 
+function loadDatasets(endpoint, type, callback, error) {
```

Comment:
please merge this function with the generic loadJSON function (maybe the you should always call `callback(response)` )